incubator/openfaas/README.md

```diff
@@ -0,0 +1,68 @@
+## Deploy OpenFaaS via Helm
+
+**Note:** You must also pass `--set rbac=false` if your cluster is not configured with role-based access control. For further information, see [here](https://kubernetes.io/docs/admin/authorization/rbac/).
+
+---
+
+Clone the faas-netes repo including the helm chart:
+
+```
+$ git clone https://github.com/openfaas/faas-netes && \
+  cd faas-netes
+```
+
+To use defaults including the `default` Kubernetes namespace (recommended)
+
+```
+$ helm upgrade --install --debug --reset-values --set async=false openfaas openfaas/
```

Comment:
This command should be updated for the incubator chart location. Same with some other commands in this file.

Reply:
Do you have the full location?